examples/rendezvous-register/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "rendezvous-register"
```

Comment:
I think for rendezvous, it makes sense to have one crate "rendezvous" with three binaries as `src/bin/{register,server,discover}.rs`.